CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+<a name="v0.36.0"></a>
+## [v0.36.0] - 2020-08-14
+### Features
+- **logs:** Implement Log API
```

Comment:
Was this an automated change? Our CHANGELOG is updated via automation, so if this was manually added, we can remove it here.